src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -145,18 +145,21 @@ export class DiscussionThread {
 
     const newComment: IComment = Utils.cloneDeep(commentStreamMessage.data);
     if (commentStreamMessage.name === "commentDelete") {
-      if (this.threadDictionary[newComment._id]) delete this.threadDictionary[newComment._id];
+      if (this.threadDictionary[newComment._id]) {
+        this.threadComments = this.threadComments.filter(comment => comment._id !== newComment._id);
+        this.threadDictionary = this.arrayToDictionary(this.threadComments);
+      }
     } else {
       const key = await this.discussionsService.importKey(this.discussionId);
 
-      newComment.text = (newComment.metadata.encrypted) ?
-        await this.discussionsService.decryptWithAES(
-          newComment.metadata.encrypted,
-          newComment.metadata.iv,
-          key,
-        ) :
-        newComment.text;
-
+      const decrypted = await this.discussionsService.decryptWithAES(
+        newComment.metadata.encrypted,
+        newComment.metadata.iv,
+        key,
+      );
+      if (newComment.metadata.encrypted) {
+        newComment.text = decrypted;
+      }
```

Comment:
Changed this block, because after updating the code, an eslint error appeared (sth with race-condition).

New way: _always_ decrypts, and _then_ checks if we have encrypted content.
Old way: only decrypt _after_ encyrpt check

Imo: Nothing changes, as we decrypt everything anyway.